docs/docs/features/recovery.md

```diff
@@ -1,14 +1,14 @@
 # State and recovery
 
-Persistent storage for confidential applications in the cloud requires a bit of attention.
+Persistent storage for confidential applications in the cloud requires attention.
 If an application should be able to restart without manual intervention, it needs a way to automatically and securely obtain a secret to decrypt its state.
 
 The SGX programming model considers a single, local application running on a specific CPU.
-In this case, the application can use the SGX seal key as its root secret.
+The application can use the SGX seal key as its root secret.
 However, this binds the application and its state to the physical machine because seal keys are unique to a single CPU.
-In sum, this means that the usual SGX programming model isn't suited for virtual environments or distributed applications.
+The usual SGX programming model isn't suited for virtual environments or distributed applications.
```

Comment:
nit: some phrase/word (like "in sum") that signals that this is the conclusion makes it easier to read IMO